src/ngraph/runtime/cpu/pass/cpu_fusion.cpp

```diff
@@ -1299,3 +1302,51 @@ void ngraph::runtime::cpu::pass::CPUFusion::construct_sigmoid_multiply()
     auto m = std::make_shared<ngraph::pattern::Matcher>(elem_mul, callback);
     this->add_matcher(m);
 }
+
+void ngraph::runtime::cpu::pass::CPUFusion::construct_bounded_relu()
+{
+    auto relu_input = std::make_shared<pattern::op::Label>(element::f32, Shape{});
+    auto relu = std::make_shared<op::Relu>(relu_input);
+    auto iconst1 = op::Constant::create(element::f32, Shape{}, {1});
+    auto alpha = std::make_shared<pattern::op::Label>(iconst1);
+    auto min = std::make_shared<op::Minimum>(relu, alpha);
+
+    pattern::graph_rewrite_callback callback = [relu_input, alpha](pattern::Matcher& m) {
+        NGRAPH_DEBUG << "In a callback for construct_bounded_relu against "
+                     << m.get_match_root()->get_name();
+
+        if (m.get_match_root()->get_element_type() != element::f32)
+        {
+            NGRAPH_DEBUG << "mpattern = " << m.get_match_root()->get_name()
+                         << " type is not float!";
+            return false;
+        }
+        auto pattern_map = m.get_pattern_map();
+        if (!std::dynamic_pointer_cast<op::Constant>(pattern_map[alpha]))
+        {
+            throw ngraph_error("alpha must be constant for bounded relu");
+        }
+
+        //we wont fuse if the alpha and the Relu output element type are not same
+        if (pattern_map[alpha]->get_element_type() != pattern_map[relu_input]->get_element_type())
+        {
+            return false;
+        }
+        if (pattern_map[alpha]->get_shape() != pattern_map[relu_input]->get_shape())
+        {
+            return false;
+        }
+
+        auto alpha_const_op = std::dynamic_pointer_cast<op::Constant>(pattern_map[alpha]);
+        float alpha_val = *(static_cast<float const*>(alpha_const_op->get_data_ptr()));
```

Comment:
we are only checking the very first element of a constant. In some obscure corner case, other elements might not necessarily be equal to the first element. Do we care about such a case? @pruthviIntel @jbobba ?